fix: [docs] reconfigured docs pages

swap-aws opened the pull request:
## Description
Changes to docs pages/structure

**Does this PR introduce a breaking change?**  
_(What changes might users need to make in their application due to this PR?)_
<!-- Describe the breaking change and reasoning -->
<!-- Does this require a migration guide? -->

## Related Issues/Discussion
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->

* **What environment was this tested on?**
<!-- Please run `pnpm run prerequisite-check` and paste output here -->

## Screenshots (if appropriate)
<!-- Add screenshots here -->

## PR Checklist

* [x] Have you added/updated documentation?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran build and tests with your changes locally?

**IMPORTANT:** Please review the [CONTRIBUTING.md](../CONTRIBUTING.md) file for detailed contributing guidelines.

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

swap-aws committed:
docs: reconfiguring docs structure

martenpayne committed:
Revert "docs: reconfiguring docs structure"

This reverts commit 8c583488b7242368ae7cda93c2348cab674cb771.

swap-aws committed:
docs: reconfigured docs pages

swap-aws committed:
docs: reconfigured docs pages

swap-aws committed:
docs: fix issues

swap-aws committed:
Merge branch 'mainline' into so-docs-home

swap-aws committed:
docs: fixed docs structure

martenpayne reviewed (commented):
Looking like good progress
- the overview landing grid is good…would it be possible to tighten up the spacing a little?
- having navigated to a documentation section, it's not obvious how to get back to that landing page
- the refactored sections are good

swap-aws commented:
> Looking like good progress
> 
>     * the overview landing grid is good…would it be possible to tighten up the spacing a little?
> 
>     * having navigated to a documentation section, it's not obvious how to get back to that landing page
> 
>     * the refactored sections are good

- Tightened up spacing and adjusted font sizes and heights
- Added a new Home menu item that gets customers back to landing page

swap-aws committed:
docs: changes from feedback

swap-aws committed:
docs: added logo image and WIP admonition

swap-aws committed:
docs: revert logo
